scripts/custom_sql_query/run_custom_query.py

```diff
@@ -197,6 +197,8 @@ def main(
     expected_before=None,
     expected_after=None,
 ):
+    sql_file = f"/function/{sql_file}"
```

Comment:
This is a fix to our custom sql script that i thought i might as well add to this PR. It's not part of the functionality of this PR